tests/Google/ClientTest.php

```diff
@@ -684,4 +686,23 @@ public function testTokenCallback()
 
     $this->assertTrue($called);
   }
+
+  public function testExecuteWithFormat()
+  {
+    $this->onlyGuzzle6();
+
+    $client = new Google_Client([
+      'api_format_v2' => true
+    ]);
+
+    $guzzle = $this->prophesize('Google_Client');
```

Comment:
Can we use `$this->getMock()` here for consistency with the rest of the tests?

Reply:
done

Reply:
Thanks!